proof/publisher.go

```diff
@@ -267,10 +267,9 @@ func (pub *publisher) process() {
 
 	// 64 bit nonce (8 bytes)
 	randomBytes := make([]byte, 8)
-	rand.Read(randomBytes)
+	_, _ = rand.Read(randomBytes)
```

Comment:
ignore error is bad; can random fail?
how about count can that be ignored also? (in bitmarkd some code check some code does not - need to decide a consistent policy)

Reply:
I am not sure when `rand.Read` fails, but I will add error checking

Reply:
there is another form using ReadFull and not sure if this is better (it was in the godoc sample code)
~~~
_, err := io.ReadFull(rand.Reader, byteSlice)
~~~
not sure how this differs

Reply:
I changed the cli to use plain rand.Read so let's stay with that